.changeset/mean-items-retire.md

```
@@ -0,0 +1,36 @@
+---
+"@clerk/backend": minor
+---
+
+Improve `subject` property handling for machine auth objects.
```

markdownlint-cli2 (0.17.2)

First line in a file should be a top-level heading
null

(MD041, first-line-heading, first-line-h1)